### ISSUE TITLE
Renovate and similar only update if binary or dependency is available for arm64 AND amd64

### ISSUE THREAD

**Issue by Maleware** [stackabletech/issues#487]
ToDo's:

- Look through https://github.com/stackabletech/docker-images to find some renovate dependencies others then ours.
- Look through https://github.com/stackabletech/operator-templating to find some dependencies

**Comment by Maleware** [comment]
moved to development as Refinement was done already. Started with doing it.

**Comment by Maleware** [comment]
Operators look good. Both pipeline and images are well reviewed.

I've didn't change anything on renovate, just ensured that the actual dependencies are on arm64 and x86.

**Comment by Maleware** [comment]
One thing:

In `docker-images` we have a few pipelines. It appears that none of them are watched by renovate at this moment. It seems this is not on purpose.

I will have a talk with @lfrancke once he is back in the drivers seat. Till then I'll move this to track.

**Comment by Maleware** [comment]
After some clarifying, testing and looking at the results of renovate runs, it looks pretty good with renovate and multi architecture.

It is watching all dependencies necessary and updates correct and according. However, this is not due to renovate being configured this way, it's the arm environment being at a level now with x86.

I've checked and tested a few repos ( not all ) and wasn't able to find a flaw in terms of MA.

This said, there is no possibility ( according to google and docs ) to configure renovate to check first if dependencies are multi arch available. It's save to assume that if a dependency is multi-arch, the next release will be too.

Nothing to do and looks good.
Closing the issue as done.
